dev/PushNotifications/externs.h

```diff
@@ -25,7 +25,10 @@ struct ChannelDetails
     winrt::Windows::Foundation::DateTime channelExpiryTime;
 };
 
-inline HRESULT GetCurrentProcessPath(wil::unique_cotaskmem_string& processName)
+inline std::wstring GetCurrentProcessPath()
 {
-    return wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, processName);
+    std::wstring processPath{};
+    THROW_IF_FAILED(wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, processPath));
```

Comment:
How is this even working? What is the third argument for this api? a wstring?? I would be very surprised if so.

Reply:
I think it needs to be wil::unique_cotaskmem_string. Otherwise it may not allocate\deallocate memory correctly

Reply:
3rd argument is string_type.

Reply:
https://github.com/microsoft/wil/blob/23fca640eae099f9120545483ae7d5d0a98a6fa5/include/wil/win32_helpers.h#L200

maker.release() here returns a std::wstring  so we should be good :)